docs/pages/workflow/expo-cli.md

```diff
@@ -391,15 +415,14 @@ Alias: `expo update`
 
 Alias: `expo ph`
 
-| Option         | Description             |
-| -------------- | ----------------------- |
+| Option                                 | Description                                                                                          |
+| -------------------------------------- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- |
 | `-c, --release-channel [channel-name]` | Filter by release channel. If this flag is not included, the most recent publications will be shown. |
-| `--count [number-of-logs]` | Number of logs to view, maximum 100, default 5. |
-| `-p, --platform [ios|android]` | Filter by platform, android or ios. Defaults to both platforms. |
-| `-s, --sdk-version [version]` | Filter by SDK version e.g. 35.0.0 |
-| `-r, --raw` | Produce some raw output. |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
+| `--count [number-of-logs]`             | Number of logs to view, maximum 100, default 5.                                                      |
+| `-p, --platform [ios                   | android]`                                                                                            | Filter by platform, android or ios. Defaults to both platforms. |
```

Comment:
md auto-formatting breaks because of `--platform [ios|android]` (the `|` is understood as table col end)

Reply:
https://github.com/expo/expo-cli/pull/3517